webapp/home/labs/proteomics/conversion_modification.yml

```diff
@@ -0,0 +1,41 @@
+id: conversion_modification
+title: MS file conversion
+tabs:
+  - id: tools
+    title: Tools
+    heading_html: >
+      Some example tools are listed here: you can also search for more in the full tool panel to the left.
+    content:
+      - title_html: "<code>msconvert</code>"
+        description_html: >
+          <p>
+            Convert and/or filter mass spectrometry files.
+          </p>
+        inputs:
+          - datatypes:
+              - thermo.raw
+              - mzML
+              - mzXML
+              - raw
+              - wiff
+              - wiff.tar
+              - agilentbrukeryep.d.tar
+              - agilentmasshunter.d.tar
+              - brukerbaf.d.tar
+              - brukertdf.d.tar
+              - watersmasslynx.raw.tar
+            label: Input MS data
+        button_link: "{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/msconvert/msconvert/"
+      - title_html: "<code>Thermo RAW file converter</code>"
+        description_html: >
+          <p>
+            Thermo RAW file converter.
```

Comment:
Suggestion - could you add to the description a list of output datatypes for this conversion? Like `<code>wiff</code>`, `<code>bruker</code>` etc. to make it immediately obvious to a user whether this is useful to them?

Reply:
Added a new commit with outputs.